doc/visual-programming/source/widgets/unsupervised/louvainclustering.rst

```diff
@@ -0,0 +1,38 @@
+Louvain Clustering
+=======
+
+Groups items using the Louvain clustering algorithm.
+
+Inputs
+    Data
+        input dataset
+
+Outputs
+    Data
+        dataset with cluster index as a class attribute
+    Graph (with the Network addon)
+		the weighted k-nearest neighbor graph
+
+
+The widget first converts the input data into a k-nearest neighbor graph. To preserve the notions of distance, the Jaccard index for the number of shared neighbors is used to weight the edges. Finally, a `modularity optimization <https://en.wikipedia.org/wiki/Louvain_Modularity>`_ community detection algorithm is applied to the graph to retrieve clusters of highly interconnected nodes. The widget outputs a new dataset in which the cluster index is used as a meta attribute.
+
+
+.. figure:: images/Louvain-stamped.png
+
+1. PCA processing is typically be applied to the original data to remove noise.
+2. The distance metric is used for finding specified number of nearest
+   neighbors.
+3. The number of nearest neighbors to use to form the KNN graph.
+4. Resolution is a parameter for the Louvain community detection algorithm that
+   affects the size of the recovered clusters. Smaller resolutions recover
+   smaller, and therefore a larger number of clusters, and conversely, larger
+   values recover clusters containing more data points.
+5. When *Apply Automatically* is ticked, the widget will automatically
+   communicate all changes. Alternatively, click *Apply*.
+
```

Comment:
At the bottom it is great to provide a minimum workflow for the widget. Or a cool way how to use it, preferably on a simple data set that exists in Orange.